final_prep/README.md

```diff
@@ -44,24 +44,27 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+- <img width="1153" alt="Screen Shot 2022-02-13 at 4 29 52 PM" src="https://user-images.githubusercontent.com/98700876/153780239-124c9099-3d3a-4fb6-a9f3-5dba67741337.png">
+
+- <img width="1153" alt="Screen Shot 2022-02-13 at 4 30 01 PM" src="https://user-images.githubusercontent.com/98700876/153780229-49dc927a-2e36-45c6-b915-0b019afb633c.png">
+
+- <img width="1153" alt="Screen Shot 2022-02-13 at 4 30 13 PM" src="https://user-images.githubusercontent.com/98700876/153780188-3a4a06d1-8275-43dd-a59d-de3d51818fc3.png">
+
 
 ## Mentorship Prep
 
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 
 - [ ] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
 - [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+  - [ ] Add link to your gist here: https://gist.github.com/Deming-Matt/58f8bfc6ff0b29380ef198588945bd24 
```

Comment:
Good start here. Your mentor will be an important person to lean on especially in Mod 1. I'd encourage you to go a little deeper on some of these questions when you get introduced to your mentor in a few weeks.